sdk/include/stdlib.h

```diff
@@ -4,10 +4,37 @@
 #pragma once
 
 #include <cdefs.h>
+#include <compartment-macros.h>
 #include <stddef.h>
 #include <stdint.h>
 #include <timeout.h>
 
+#ifndef MALLOC_DEFAULT_QUOTA
+#	define MALLOC_DEFAULT_QUOTA 4096
+#endif
+struct MallocCapability
+{
+	size_t    quota;
+	size_t    unused;
+	uintptr_t reserved[2];
```

Comment:
I might reserve more space; we might want to have these...
- know a small identifier for themselves (another `uint16_t`),
- be intrusively indexed on that identifier by a list or tree or something (2 or 3 `uintptr_t`s, a la `MChunk` or `TChunk`)
- have a hierarchical relationship amongst themselves (perhaps as a rose tree with children encoded as a ds::linked_list ring... another 5 `uintptr_t`s)

I don't think we imagine that there will be too many of these, so making them kind of hefty is probably not the worst fate?

Reply:
I was in two minds about whether to reserve state at all.  Until we do an ABI-stable release, I can add fields at will later.